src/commcare_cloud/ansible/roles/citusdb/tasks/main.yml

```diff
@@ -32,7 +32,7 @@
      postgresql_shared_preload_libraries:
        - 'citus'
        - 'pg_stat_statements'
-     postgresql_custom_settings: "{{ [\"citus.use_secondary_nodes = 'always'\"] if is_pg_standby else []}}"
+     postgresql_custom_settings: "{{ [\"citus.use_secondary_nodes = 'always'\"] if is_pg_standby and 'citusdb_master' in groups else []}}"
```

Comment:
i don't why but this evaluation condition is not working as it should be.

Reply:
is there an error?

Reply:
No error . it evaluates to true even for the citusworkerstandby node.

Reply:
```suggestion
     postgresql_custom_settings: "{{ [\"citus.use_secondary_nodes = 'always'\"] if is_pg_standby and inventory_hostname in groups['citusdb_master']|default([]) else []}}"
```